src/client.js

```diff
@@ -134,6 +134,18 @@ class PureCloudWebrtcSdk extends WildEmitter {
   disconnect (id) {
     this._streamingConnection.disconnect();
   }
+
+  _refreshTurnServers () {
+    this._streamingConnection.requestExternalServices('turn', (err, services) => {
+      if (err) {
+        const err = 'PureCloud SDK failed to update TURN credentials. The application should be restarted to ensure connectivity is maintained.';
```

Comment:
Recommend calling this something different from the `err` from the function to avoid confusion / variable shadowing